src/components/MultiHopTrade/hooks/useGetTradeQuotes/useGetTradeQuotes.tsx

```diff
@@ -291,25 +255,120 @@ export const useGetTradeQuotes = () => {
     return () => clearInterval(interval)
   }, [])
 
-  useGetTradeQuoteQuery(tradeQuoteInput, {
-    skip: !shouldRefetchTradeQuotes,
-    pollingInterval: hasFocus ? GET_TRADE_QUOTE_POLLING_INTERVAL : undefined,
-    /*
-    If we don't refresh on arg change might select a cached result with an old "started_at" timestamp
-    We can remove refetchOnMountOrArgChange if we want to make better use of the cache, and we have a better way to select from the cache.
-    */
-    refetchOnMountOrArgChange: true,
-  })
+  const skip = !shouldRefetchTradeQuotes
+  const pollingInterval = hasFocus ? GET_TRADE_QUOTE_POLLING_INTERVAL : undefined
+
+  const cowSwapQuoteMeta = useGetSwapperTradeQuote(
+    SwapperName.CowSwap,
+    tradeQuoteInput,
+    skip,
+    pollingInterval,
```

Comment:
We could abstract these three into common args to improve readbility if we want to

Reply:
nice 6af8a5674831168abbd5702e260caa006cd4e828